salt/modules/win_wusa.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+'''
+Microsoft Update files management via wusa.exe
+
+:maintainer:    Thomas Lemarchand
+:platform:      Windows
+:depends:       PowerShell
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+
+# Import salt libs
+import salt.utils.platform
+
+log = logging.getLogger(__name__)
+
+# Define the module's virtual name
+__virtualname__ = 'win_wusa'
+
+
+def __virtual__():
+    '''
+    Load only on Windows
+    '''
+    if not salt.utils.platform.is_windows():
+        return False, 'Only available on Windows systems'
+
+    powershell_info = __salt__['cmd.shell_info']('powershell', True)
+    if not powershell_info['installed']:
+        return False, 'PowerShell not available'
+
+    return __virtualname__
+
+
+def is_installed(kb):
+
+    get_hotfix_result = __salt__['cmd.powershell_all']('Get-HotFix -Id {0}'.format(kb), ignore_retcode=True)
+
+    return get_hotfix_result['retcode'] == 0
+
+
+def install(path):
+
+    return __salt__['cmd.run_all']('wusa.exe {0} /quiet /norestart'.format(path), ignore_retcode=True)
+
+
+def uninstall(kb):
```

Comment:
please change `uninstall `to `remove `to match the package management modules.

Reply:
I used "uninstall" to match the win_wua module (https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.win_wua.html#salt.modules.win_wua.uninstall), which is closer to our module than package management.